tests/integration/test_create_token.py

```diff
@@ -0,0 +1,173 @@
+import uuid
+
+from app.authentication.auth_payload_versions import AuthPayloadVersion
+from app.authentication.authenticator import decrypt_token
+from tests.integration.app_context_test_case import AppContextTestCase
+from tests.integration.create_token import (
+    ACCOUNT_SERVICE_URL,
+    PAYLOAD_V2_SOCIAL,
+    PAYLOAD_V2_SUPPLEMENTARY_DATA,
+)
+from tests.integration.integration_test_case import IntegrationTestCase
+
+test_parameters = [
+    {
+        "schema": "test_metadata_routing.json",
+        "theme": "default",
+        "additional_payload": {
+            "flag_1": 123,
+            "period_id": "abc",
+            "link": "https://example.com",
+        },
+        "payload": {
```

Comment:
This can just be:
```suggestion
        "payload": PAYLOAD_V2_BUSINESS,
```
Same applies to social below

Reply:
Updated this, I also move `test_parameters` into the test it's used by to have consistency in the file as the last test does the same